source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ACRContentStackView.h

```diff
@@ -0,0 +1,22 @@
+//
+//  ACRContentStackView
+//  ACRContentStackView.h
+//
+//  Copyright © 2017 Microsoft. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@interface ACRContentStackView: UIView
```

Comment:
I'm not sure I understand your point. A UIStackView can definitely specify its own frame and constraints.

---
In reply to: [128332929](https://github.com/Microsoft/AdaptiveCards/pull/488#discussion_r128332929) [](ancestors = 128332929,128110253)